vdirsyncer/storage/ldap.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+import logging
+import ldap3
+
+from .base import Item, Storage
+from .. import exceptions
+
+ldap_logger = logging.getLogger(__name__)
+
+
+class LDAPStorage(Storage):
+
+    __doc__ = '''
```

Comment:
Thanks for the quick reply.

I'm now using vobject which is clearly cleaner. Thanks for the hint.
I've also added support for title and fax.

I changed the default filter to one that make sense since `vobject`
clearly states that `F` and `FN` are mandatory fields. On some dirty
ldap directory `sn` or `givenName` could be missing and I'm not sure how to
handle it cleanly without filtering it. Maybe just provide the available
`family` or `givenname` for `N`.

About tests, I'm not sure how we could test it against a real ldap
server. I don't know any open ldap server but I can check.

Note that the ldap attributes I'm using are from Active Directory. Any
other ldap directory could also use it but this is not guaranteed. The
most problematic attribute is `whenChanged`. I've never seen it on other
classic ldap directory and it's used for etag.

-- 
Paul Fariello

PGP: 0x672CDD2031AAF49B